Animal Competition/Java/Game.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+class Game {
+  void startGame(){
+    String greeting = "Hello! Welcome to (insert name here)!";
+
+    System.out.println(greeting);
+
+    Scanner playerNumber = new Scanner(System.in);
+
+    System.out.println("Press 1 for singleplayer. Press 2 for two players.");
+    
+    int numberOfPlayers = playerNumber.nextInt();
+      // if one player is playing
+    if (numberOfPlayers == 1) {
+      System.out.println("Welcome!");
+      
+      // makes a new scanner to scan for user input
+      Scanner userNameInput = new Scanner(System.in);
```

Comment:
Notice that there's a lot of repeated code just for getting the name of the player from user input?  You should refactor that into a function like:
```suggestion
      var userName = this.getPlayerName("");
      System.out.println(String.format("Welcome %s", userName));
```
```java
  // Prompts the user to end their name and returns it.
  String getPlayerName(String prefix) {
    var userNameInput = new Scanner(System.in);
    System.out.println("%sPlease enter your Username:", prefix.length() > 0 ? String.format("%s ", prefix) : "");
    var userName = userNameInput.nextLine();
    return userName;
  }
```